src/pudl/package_data/eia923/column_maps/emissions_control.csv

```diff
@@ -0,0 +1,24 @@
+pm_removal_efficiency_100pct_load,pm_tested_efficiency_removal_at_100_load,pm_tested_efficiency_removal_at_100_load,pm_tested_efficiency_removal_at_100_load,pm_tested_efficiency_removal_at_100_load,pm_tested_efficiency_removal_at_100_load,pm_tested_efficiency_removal_at_100_load,pm_tested_efficiency_removal_at_100_load,pm_tested_efficiency_removal_at_100_load,pm_tested_efficiency_removal_at_100_load,pm_tested_efficiency_removal_at_100_load
+pm_test_date,pm_test_date,pm_test_date,pm_test_date,pm_test_date,pm_test_date,pm_test_date,pm_test_date,pm_test_date,pm_test_date,pm_test_date
+so2_removal_efficiency_annual,so2_removal_efficiency_rate_at_annual_operating_factor,so2_removal_efficiency_rate_at_annual_operating_factor,so2_removal_efficiency_rate_at_annual_operating_factor,so2_removal_efficiency_rate_at_annual_operating_factor,so2_removal_efficiency_rate_at_annual_operating_factor,so2_removal_efficiency_rate_at_annual_operating_factor,so2_removal_efficiency_rate_at_annual_operating_factor,so2_removal_efficiency_rate_at_annual_operating_factor,so2_removal_efficiency_rate_at_annual_operating_factor,so2_removal_efficiency_rate_at_annual_operating_factor
+so2_removal_efficiency_100pct_load,so2_tested_efficiency_removal_at_100_load,so2_tested_efficiency_removal_at_100_load,so2_tested_efficiency_removal_at_100_load,so2_tested_efficiency_removal_at_100_load,so2_tested_efficiency_removal_at_100_load,so2_tested_efficiency_removal_at_100_load,so2_tested_efficiency_removal_at_100_load,so2_tested_efficiency_removal_at_100_load,so2_tested_efficiency_removal_at_100_load,so2_tested_efficiency_removal_at_100_load
+so2_test_date,so2_test_date,so2_test_date,so2_test_date,so2_test_date,so2_test_date,so2_test_date,so2_test_date,so2_test_date,so2_test_date,so2_test_date
+fgd_sorbent_consumption_1000_tons,fgd_sorbent_quantity_thousand_tons,fgd_sorbent_quantity_thousand_tons,fgd_sorbent_quantity_thousand_tons,fgd_sorbent_quantity_thousand_tons,fgd_sorbent_quantity_thousand_tons,fgd_sorbent_quantity_thousand_tons,fgd_sorbent_quantity_thousand_tons,fgd_sorbent_quantity_thousand_tons,fgd_sorbent_quantity_thousand_tons,fgd_sorbent_quantity_thousand_tons
+fgd_electricity_consumption_mwh,fgd_electricity_consumption_megawatthours,fgd_electricity_consumption_megawatthours,fgd_electricity_consumption_megawatthours,fgd_electricity_consumption_megawatthours,fgd_electricity_consumption_megawatthours,fgd_electricity_consumption_megawatthours,fgd_electricity_consumption_megawatthours,fgd_electricity_consumption_megawatthours,fgd_electricity_consumption_megawatthours,fgd_electricity_consumption_megawatthours
+hg_removal_efficiency,mercury_removal_efficiency,mercury_removal_efficiency,mercury_removal_efficiency,mercury_removal_efficiency,mercury_removal_efficiency,mercury_removal_efficiency,mercury_removal_efficiency,mercury_removal_efficiency,mercury_removal_efficiency,mercury_removal_efficiency
```

Comment:
Would be nice to have consistency and use either mercury or hg in our codes. So far, we've been using mercury (e.g. `compliance_year_mercury`).